<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messenger_Main</class>
 <widget class="QMainWindow" name="Messenger_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_Contact_Selected">
          <property name="styleSheet">
           <string notr="true">font-size: 32px;
 font-weight: 500;
    line-height: 42px;
    text-align: center;
    color: #ffffff;
    letter-spacing: 0.5px;</string>
          </property>
          <property name="text">
           <string>Contact selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_Messages"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_message_to_send"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Send_Message">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Export_PDF">
          <property name="text">
           <string>Export PDF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QWidget" name="" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-size: 32px;
 font-weight: 500;
    line-height: 42px;
    text-align: center;
    color: #ffffff;
    letter-spacing: 0.5px;</string>
         </property>
         <property name="text">
          <string>Contacts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_Contacts"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_Group_Message">
           <property name="text">
            <string>Group Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Add_Contact">
           <property name="text">
            <string>Add Contact</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Add_Contact"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
